client/style/style.css

```diff
@@ -66,6 +66,23 @@ footer form, .footer-item {
 
 .footer-item { margin-left: 8px; }
 
+.incremental-search {
+  position: absolute;
+  background-color: white;
+  color: black;
+  display: inline-block;
+  max-height: 300px;
+  padding-top: 5px;
+  padding-right: 10px;
+  padding-left: 10px;
+  box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.4);
+  outline: none;
```

Comment:
adding `overflow: hidden;` should help with text spilling out of the search results.

Reply:
I added `overflow-y: auto` (to allow a scrollbar to be shown if and only if it is needed) and `overflow-x: hidden`.